src/hooks/sdk/docs/useMediaStreamMetrics.stories.mdx

```diff
@@ -2,19 +2,29 @@
 
 # useMediaStreamMetrics
 
-The `useMediaStreamMetrics` hook returns the available audio, video and bandwidth metrics for the current meeting session. Video metrics contains stream metrics for all attendeeIds.
+The `useMediaStreamMetrics` hook returns:
+
+- common metrics for audio stream
```

Comment:
"common metrics" is vague, are these any other than WebRTC stats? Would be good to be specific :)

Reply:
These are all WebRTC stats. It's hard to summarize what these metrics really are because they covered several different topics. I think "common" is fine and because we list the detailed metrics below. Besides we might need to expose more and more metrics in this hook. Don't think it's ideal to be very specific in the introduction section. How do you think?

Reply:
If these all are WebRTC stats, can we just say "WebRTC metrics" over "common metrics" or just "Metrics"?

Reply:
Upto you to change, no strong opinion for now.